pymultimatic/model/common.py

```diff
@@ -46,14 +46,21 @@ def active_mode(self) -> ActiveMode:
         :class:`~pymultimatic.model.system.System` for that.
         """
         mode = None
-        if self.operating_mode == OperatingModes.AUTO:
+        if (
+            self.operating_mode == OperatingModes.AUTO
+            or self.operating_mode == OperatingModes.TIME_CONTROLLED
+        ):
             if self.time_program:
                 setting = self.time_program.get_for(datetime.now())
                 if setting:
                     if setting.setting in [SettingModes.DAY, SettingModes.ON]:
-                        mode = ActiveMode(self.target_high, OperatingModes.AUTO, setting.setting)
+                        if self.operating_mode == OperatingModes.AUTO:
+                            target = self.target_high
+                        else:
+                            target = setting.target_temperature
+                        mode = ActiveMode(target, self.operating_mode, setting.setting)
```

Comment:
If I understand correctly, you can configure a **zone** to have multiple different temperatures based on the timepgoram ?

WIth multimatic, you have to configure high and low temperatures and you choose what time you want high. So this is either low or hight

Reply:
Yes, there is a setback temperature (REDUCTED) "heating/configuration/setback_temperature_setpoint" and you define the periods with a specific temperature. Undefined periods are set to REDUCTED.

Reply:
Yes, you can have a target temperature of 21.5 from 5am to 9 am, 21.0 from 4pm to 10pm, the gaps are filled with the night target temp.